stl/inc/print

```diff
@@ -24,15 +24,128 @@ _STL_DISABLE_CLANG_WARNINGS
 
 _STD_BEGIN
 
-inline void _Print_noformat_nonunicode(FILE* const _Stream, const string_view _Str) {
-    const bool _Was_write_successful = _CSTD fwrite(_Str.data(), 1, _Str.size(), _Stream) == _Str.size();
+struct [[nodiscard]] _Stream_lock_guard {

```

Comment:
In headers of product code (but _not_ in source files), we conventionally use `_NODISCARD`.
```suggestion
struct _NODISCARD _Stream_lock_guard {
```